### PR TITLE
Add mlops/dagshub

### PR DESCRIPTION
**Pull request by simonlsk**
# Summary

Added DagsHub pack under `mlops`.
The `config.yml` file still needs to be authored.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### CODE REVIEW

File: packs/mlops/dagshub/config.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - ml
+  - ai
+  - dagshub
+  - machine-learning
+  - machine learning
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - dagshub-mlops
```
Comment (polfliet) [inline packs/mlops/dagshub/config.yml:35]: This installPlan will have to be created, similar to the example for Algorithmia here https://github.com/newrelic/newrelic-quickstarts/blob/main/install/mlops/algorithmia/install.yml.
Can you add that installPlan to the PR?

File: packs/mlops/dagshub/dashboards/dagshub.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          },
+          "title": "",
+          "rawConfiguration": {
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 3240610,
+                "query": "SELECT latest(loss) FROM Metric TIMESERIES SINCE 1800 seconds ago"
```
Comment (polfliet) [inline packs/mlops/dagshub/dashboards/dagshub.json:27]: (disclaimer: I don't know anything about DagsHub)
Will this `loss` event be available for every user that sets up DagsHub?
The goal of the dashboard is that it should work for every user who tries this out.

Are there other charts that could be added to make this a compelling dashboard?
Or does this really depend on the customer's use case? Maybe in that case, would it make sense to use a markdown widget on the dashboard and provide some more info?

**Review by polfliet — CHANGES_REQUESTED** [review]
Hey @simonlsk! Thanks for your contribution!

**Comment by jpvajda** [comment]
I am closing this PR due to the age and no response from the contributor. If you'd like to resubmit this, we can re-open when you have time to resolve the feedback @simonlsk  Thank you!

**Comment by simonlsk** [comment]
@jpvajda I took care of the required changes. Can you reopen?

**Comment by jpvajda** [comment]
@simonlsk Thanks I re-opened the PR!

**Comment by jpvajda** [comment]
@simonlsk this is failing the schema compliance check could you take a [look at the erro](https://github.com/newrelic/newrelic-quickstarts/runs/4229216714?check_suite_focus=true)r and try to resolve?

File: quickstarts/mlops/dagshub/config.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+id:
```
Comment (rudouglas) [inline quickstarts/mlops/dagshub/config.yml:1]:
```suggestion
```

The id will be autogenerated so you can remove this line

File: quickstarts/mlops/dagshub/config.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+authors:
+  - New Relic
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - mlops
+  - ml
+  - ai
+  - dagshub
+  - machine-learning
```
Comment (rudouglas) [inline quickstarts/mlops/dagshub/config.yml:29]:
```suggestion
```

**Review by rudouglas — CHANGES_REQUESTED** [review]
Just a couple of small changes

**Review by rudouglas — APPROVED** [review]

**Pull request merged**